scripts/simple_raco_execution.py

```diff
@@ -0,0 +1,87 @@
+
+'''
+A notional raco-only script for executing a MyriaL query
```

Comment:
That's the goal, but I'd like to get this refactoring into master before changing the interface.  (In hindsight, I probably should have put these changes in a separate branch.)  Maybe we can rename this file as a txt to indicate that it is just notional at this point?